Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN npm install && npm run build
 
 FROM nginx:1.24.0-alpine3.17
 
-RUN apk update && apk upgrade
+RUN apk update && apk upgrade && apk add bash
```

Comment:
It is not advised to upgrade and add packages in a dockerfile as it can pull in different dependency licenses than the original base image has. Also why do yo need bash for this image?

Reply:
bash is required to run script but i will removed it and update change.

Reply:
Removed apk upgrade cmd.